### ISSUE TITLE
EntityResolution service should be more secured by RBAC casbin privileges

### ISSUE THREAD

**Issue by jakedoublev** [opentdf/platform#1046]
### Background 

The current RBAC Casbin `defaultPolicy` gives ERS route access to the `standard` role. This route should be of higher privilege to avoid an authenticated user querying another entity's idP value entity resolution.

https://github.com/opentdf/platform/blob/db4f06fdb9314747d9a95a5a09f974d86a1f0f29/service/internal/auth/casbin.go#L85

### Acceptance Criteria

1. ERS is protected by a higher privilege role
2. (optional) roles are enhanced to include a new intra-platform service or PEP role that is intended to be distinct from administrators or workflows

**Comment by jrschumacher** [comment]
@elizabethhealy @jakedoublev is this done with the reduced complexity of our default authz policy?

**Comment by elizabethhealy** [comment]
@jrschumacher yes it is restricted to admin with the current casbin policy https://github.com/opentdf/platform/blob/main/service/internal/auth/casbin_policy.csv
